api/alembic/versions/2b3755392ad8_percent_grass_curing.py

```diff
@@ -0,0 +1,41 @@
+"""Percent grass curing
+
+Revision ID: 2b3755392ad8
+Revises: 403586c146ae
+Create Date: 2024-02-01 16:45:05.914743
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from app.db.models.common import TZTimeStamp
+
+# revision identifiers, used by Alembic.
+revision = '2b3755392ad8'
+down_revision = '403586c146ae'
```

Comment:
My migration beat you to the punch 😏. I think the `down_revision` should reference my last PR? `5845f568a975`

Reply:
Good catch! Thanks for the reminder. I'll have to update, tear down this PR and re-deploy.